spec/spec_helper.rb

```diff
@@ -4,7 +4,20 @@
 
 require 'combustion'
 
+Dir[Rails.root.join('spec/support/*.rb')].each { |f| require f }
+
 Combustion.initialize!
 
 RSpec.configure do |config|
+  config.include SpecMacros
+end
+
+class StubbedMappingService
```

Comment:
What about moving this to a file under `spec/support`? Or even better, since we're using this service on all (?) uploader specs, should we generalize this and move it to dradis-plugins?

Reply:
Instead of adding a dependency on dradis-plugins and  running the specs from within the gem, I updated them to be run from within the main app so we don't need this anymore